### PR TITLE
Update cron-examples-reference.md

### PR DESCRIPTION
**Pull request by datfinesoul**
Incorrect `*` character that should be `-`

## Purpose of this pull request

This pull request (PR) ... fixes a typo

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/installed-collectors/sources/script-source/cron-examples-reference.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ There are several special characters that are used to specify values:
 |:--|:--|:--|
 | \* | All values. | `*` in the minute field means every minute. |
 | ? | No specific value in the day of month and day of week fields. | `?` specifies a value in one field, but not the other. |
-| \* | A range. | `10-12` in the hour field means the script will run at 10, 11 and 12 (noon). |
+| - | A range. | `10-12` in the hour field means the script will run at 10, 11 and 12 (noon). |
```
Comment (JV0812) [inline docs/send-data/installed-collectors/sources/script-source/cron-examples-reference.md:42]:
```suggestion
| - | A range. | `10-12` in the hour field means the script will run at 10, 11, and 12 (noon). |
```

**Comment by jpipkin1** [comment]
Added @datfinesoul to the .clabot file with PR #2870.

File: docs/send-data/installed-collectors/sources/script-source/cron-examples-reference.md (hunk 1 above)
Comment (jpipkin1) [reply docs/send-data/installed-collectors/sources/script-source/cron-examples-reference.md:42]: @JV0812, I'll go ahead and commit your suggestion.

**Comment by jpipkin1** [comment]
@datfinesoul, click the "sign your name here" link in the **cla-bot** box above so we can add you to our list of contributors.

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**